### DISCUSSION TITLE
希望能自定义根目录

### DISCUSSION THREAD

**Discussion by moesnow** [messense/aliyundrive-webdav#200] [Q&A] [answered]
比如我只希望 /Video 这个文件夹下的内容通过webdav共享，放在公网默认根目录总感觉有点风险哈

**Comment by messense** [comment, marked as answer]
`--root <root>                            Root directory path [default: /]`

文档里面就写了，你的情况 `--root Video`

**Reply by messense** [reply]
OpenWrt 界面也提供了配置。

**Comment by moesnow** [comment]
不好意思 看到已经有了
“--root”

**Reply by messense** [reply]
之前就有人提了 https://github.com/messense/aliyundrive-webdav/issues/47
